### PR TITLE
Resolve FPs reported under issue 6

### PR DESCRIPTION
**Pull request by EsadCetiner**
Closes https://github.com/coreruleset/nextcloud-rule-exclusions-plugin/issues/6 
I also fixed a few additional FP I noticed while working on this issue

### CODE REVIEW

File: plugins/nextcloud-rule-exclusions-before.conf [hunk 1]
```diff
@@ -477,6 +520,19 @@ SecRule REQUEST_FILENAME "@rx /apps/contacts/api/v[0-9]/social/config/user/enabl
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
 
+# Loading Contact profile pictures
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/[^/]+/[^/]+/[0-9]+$" \
```
Comment (theseion) [inline plugins/nextcloud-rule-exclusions-before.conf:524]:
```suggestion
SecRule REQUEST_FILENAME "@rx /remote\.php/dav/addressbooks/users/(?:[^/]+/){2}[0-9]+$" \
```

File: plugins/nextcloud-rule-exclusions-before.conf [hunk 2]
```diff
@@ -517,6 +573,53 @@ SecRule REQUEST_METHOD "@streq PROPFIND" \
         "t:none,\
         ctl:ruleRemoveById=921110"
 
+# Syncing calendars
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/calendars/[^/]+/[^/]+/[^/]+\.ics$" \
```
Comment (theseion) [inline plugins/nextcloud-rule-exclusions-before.conf:577]:
```suggestion
SecRule REQUEST_FILENAME "@rx /remote\.php/dav/calendars/(?:[^/]+/){2}[^/]+\.ics$" \
```

File: plugins/nextcloud-rule-exclusions-before.conf [hunk 3]
```diff
@@ -517,6 +573,53 @@ SecRule REQUEST_METHOD "@streq PROPFIND" \
...
+SecRule REQUEST_FILENAME "@rx /apps/calendar/v[0-9]/appointment_configs$" \
+    "id:9508335,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.description"
+
+# Emptying Calendar trashbin
```
Comment (theseion) [inline plugins/nextcloud-rule-exclusions-before.conf:611]: Please comment on why you disable 200002.

**Review by theseion — CHANGES_REQUESTED** [review]

File: plugins/nextcloud-rule-exclusions-before.conf (hunk 3 above)
Comment (EsadCetiner) [reply plugins/nextcloud-rule-exclusions-before.conf]: Fixed

**Review by theseion — APPROVED** [review]

**Comment by theseion** [comment]
Considering that @AlroneRhyn has not responded in weeks and is unlikely to respond to this PR now I'll merge this.

**Comment by EsadCetiner** [comment]
fair enough, I'll open another PR if he does report any issues or if I discover issues.

**Pull request merged**